main.cpp

```diff
@@ -13,75 +13,506 @@ What you create in this project will be used as the basis of Project 5 in the co
+    3) burn fuel
+ */
+
+struct Engine
+{
+    // pistons
+    int pistons = 2;
+    // valves
+    int valves = 16;
+    // cyclinder head  manufacturer
+    std::string cyclinderHeadManufacturer = "Suzuki";
+    // quanity of oil pans
+    int oilPans = 2;
+    // engine block type
+    std::string engineBlockType = "Combustion";
+
+    // produce energy for the motorcycle to move
+    void combust(int pistons, int valves, std::string engineBlockType);
+    // lubricate itself with oil to keep the parts moving fluidly
+    void lubricate(int oilPans, bool lubricateEngine);
+    // burn fuel
+    void burnFuel(bool burnFuel = true, bool regulateFuelIntake = true);
+};
+
+/*
+Thing 7) Exhaust
+5 properties:
+    1) length in inches // double
+    2) turbocharger power // double
+    3) catalytic converters // int
+    4) muffler type // string
+    5) number of pipes // int
+3 things it can do:
+    1) increase the power of the engine using a turbocharger
+    2) reduce fumes using catalytic converters
+    3) reduce or increase the level of noise when reving the engine
+ */
+
+struct Exhaust
+{
+    // length in inches
+    double lengthinInches = 15.4;
+    // turbocharger power
+    double turboChargerPower = 4.5;
+    // catalytic converters
+    int catalyticConverters = 1;
+    // muffler type
+    std::string mufflerType = "High Supression";
+    // number of pipes
+    int pipes = 2;
+
+    // increase the power of the engine using a turbocharger
+    void turboChargeOutput(double turboChargerPower);
+    // reduce fumes using catalytic converters
+    void catalyticConversion(bool performCatalyticConversion = true);
+    // reduce or increase the level of noise when reving the engine
+    void levelOfNoise(std::string mufflerType);
+};
+
+/*
+Thing 8) Fuel Tank
+5 properties:
+    1) capacity in litres // double
+    2) the type of fuel it can hold // string
+    3) the min amount of fuel it can hold before it starts to effect performance // double
+    4) capacity sensors // int
+    5) material // string
+3 things it can do:
+    1) notify the rider when it is low on fuel
+    2) notify the rider when the wrong type of fuel has been used
+    3) hold fuel
+ */
+
+struct FuelTank
+{
+    // capacity in litres
+    double capacityL = 8.5;
+    // the type of fuel it can hold
+    std::string fuel = "unleaded";
+    // the min amount of fuel it can hold before it starts to effect performance
+    double recommendedMinFuel = 3.5;
+    // capacity sensors
+    int sensors = 4;
+    // material
+    std::string material = "steel";
+
+    // notify the rider when it is low on fuel
+    std::string lowFuel(double recommendedMinFuel = 3.5, bool fuelIsLowerThanRecommendedAmoutn = true); // return type as std::string is ok?
+    // notify the rider when the wrong type of fuel has been used
+    std::string wrongFuel(int sensors = 4, std::string fuel = "unleaded", bool wrongFuelInTank = true);
```

Comment:
none of these names contain the 'action' or verb.
change your function names to match the plain-english description
- [ ] revise